pkgs/development/libraries/raylib/default.nix

```diff
@@ -4,9 +4,15 @@
   alsaSupport ? stdenv.hostPlatform.isLinux, alsa-lib,
   pulseSupport ? stdenv.hostPlatform.isLinux, libpulseaudio,
   sharedLib ? true,
-  includeEverything ? true
+  includeEverything ? true,
+  waylandSupport
```

Comment:
We should default this to either `true` or `false`, otherwise it does not build:

```
$ nix-build https://github.com/NixOS/nixpkgs/archive/a77076fa324fd9d0c08ec0f281401d2e71e239a7.tar.gz -A raylib
error: Function called without required argument "waylandSupport" at /nix/store/i0fszgqkja2ysij8l35fiwhhz4lm5hq4-source/pkgs/development/libraries/raylib/default.nix:8
```

Reply:
Totally forgot. I'll change it to `waylandSupport ? false` to match glfw's expression.